node/config/def.go

```diff
@@ -32,6 +32,18 @@ func defCommon() Common {
 		Backup: Backup{
 			DisableMetadataLog: true,
 		},
+	}
+}
+
+func DefaultDefaultMaxFee() types.FIL {
+	return types.MustParseFIL("0.07")
```

Comment:
Are we sure that this is only touching markets related? From my naive view it seems more "global" than that.